lib/vmpooler/metrics/promstats.rb

```diff
@@ -23,9 +23,9 @@ class Promstats < Metrics
       @p_metrics = {}
 
       def initialize(logger, params = {})
-        @prefix = params['prefix'] || 'vmpooler'
-        @metrics_prefix = params['metrics_prefix'] || 'vmpooler'
-        @endpoint = params['endpoint'] || '/prometheus'
+        @prefix = params[:prefix] || 'vmpooler'
+        @prometheus_prefix = params[:prometheus_prefix] || 'vmpooler'
+        @prometheus_endpoint = params[:prometheus_endpoint] || '/prometheus'
```

Comment:
I question changing line 26 to a symbol - that will break existing configuration and does not seem to correspond to how other metrics providers are configured.

Reply:
Yeah - have fixed the test params spec issue so reverted back to the standard way.